Bottom: 3bb9b1370778dc70d5b28028a02c1dfda26266b5
Top:    f3a89013086468517a3e48453b2edab651994481
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:20:04 +0000

Fix borast not to build for non-GL enabled builds


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 3e71e68..64bbdb1 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -38,22 +38,6 @@ PCB_SRCS = \
 	box.h \
 	buffer.c \
 	buffer.h \
-	borast/borast-minimal.h \
-	borast/borastint-minimal.h \
-	borast/borast-bentley-ottmann.c \
-	borast/borast-combsort-private.h \
-	borast/borast-compiler-private.h \
-	borast/borast-fixed-private.h \
-	borast/borast-fixed-type-private.h \
-	borast/borast-freelist.c \
-	borast/borast-freelist-private.h \
-	borast/borast-malloc-private.h \
-	borast/borast-traps.c \
-	borast/borast-traps-private.h \
-	borast/borast-types-private.h \
-	borast/borast-wideint.c \
-	borast/borast-wideint-private.h \
-	borast/borast-wideint-type-private.h \
 	change.c \
 	change.h \
 	clip.c \
@@ -194,8 +178,23 @@ GL_SRCS= \
 	hid/common/hidgl.c \
 	hid/common/hidgl.h \
 	hid/common/trackball.c \
-	hid/common/trackball.h
-
+	hid/common/trackball.h \
+	borast/borast-minimal.h \
+	borast/borastint-minimal.h \
+	borast/borast-bentley-ottmann.c \
+	borast/borast-combsort-private.h \
+	borast/borast-compiler-private.h \
+	borast/borast-fixed-private.h \
+	borast/borast-fixed-type-private.h \
+	borast/borast-freelist.c \
+	borast/borast-freelist-private.h \
+	borast/borast-malloc-private.h \
+	borast/borast-traps.c \
+	borast/borast-traps-private.h \
+	borast/borast-types-private.h \
+	borast/borast-wideint.c \
+	borast/borast-wideint-private.h \
+	borast/borast-wideint-type-private.h
 
 BUILT_SOURCES = \
 	core_lists.h \
